app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -146,6 +164,9 @@ class NotificationScheduler(
     companion object {
         const val UNUSED_APP_WORK_REQUEST_TAG = "com.duckduckgo.notification.schedule"
         const val USE_OUR_APP_WORK_REQUEST_TAG = "com.duckduckgo.notification.useOurApp"
+        const val UOA_DURATION = 3L
```

Comment:
NIT: `DURATION` was confusing to me, this made me think about something like "for how long is going to be active something".

It's clear when you read this line: `.setInitialDelay(duration, unit)`, but not in other scenarios where I don't have the context of what duration means. I propose to rename it to include `Delay`, so we have the context for that val.

Reply:
👍  renamed to `UOA_DELAY_DURATION_IN_DAYS`